qualang_tools/simulator/README.md

```diff
@@ -0,0 +1,43 @@
+# Simulator tools
+This library includes tools to help simulate programs
+
+## create_simulator_controller_connections
+Creates a list of :class:`~qm.simulate.interface.ControllerConnection` objects with the Inter-controllers connections between controllers in a way which maximizes the optical links connectivity.
+For use inside the :class:`~qm.simulate.interface.SimulationConfig` for simulating the controllers` behavior.
+Can be used to also simulate part of a small cluster: For example, the following command would create all the connections
+needed to run a simulation on 'con1', 'con4', 'con5' which are part of a 9 OPX cluster: `create_simulator_controller_connections(9, [1,4,5])`
+
+### Usage example
+A simple example for a 3 OPX cluster:
+```python
+from qualang_tools.simulator import create_simulator_controller_connections
+
+with program() as prog:
+    # QUA program 
+
+qmm = QuantumMachinesManager(qop_ip, cluster_name)
```

Comment:
If you want to show an example with cluster name, then the keyword must be specified:
`qmm = QuantumMachinesManager(qop_ip, cluster_name=cluster_name)` (at least with qm-qua==1.1.6)